roles/openshift_prometheus/README.md

```diff
@@ -0,0 +1,95 @@
+OpenShift Prometheus
+====================
+
+OpenShift Prometheus Installation
+
+Requirements
+------------
+
+
+Role Variables
+--------------
+
+For default values, see [`defaults/main.yaml`](defaults/main.yaml).
+
+- `openshift_prometheus_state`: present - install/update. absent - uninstall.
+
+- `openshift_prometheus_namespace`: project (i.e. namespace) where the components will be
+  deployed.
+
+- `openshift_prometheus_replicas`: The number of replicas for prometheus deployment.
+
+- `openshift_prometheus_node_selector`: Selector for the nodes prometheus will be deployed on.
+
+- `openshift_prometheus_image_<COMPONENT>`: specify image for the component 
+
+## Storage related variables
```

Comment:
I did not find oauth-proxy pv ansible parameters in  openshift-ansible/roles/openshift_prometheus/defaults/main.yaml, is these parameters missed/

Reply:
We don't create a PV for oauth-proxy. we use the prometheus mounts for data, and local /etc for secrets.

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/templates/prometheus_deployment.j2#L64

Same for alerts:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/templates/prometheus_deployment.j2#L136